astropy/utils/data.py

```diff
@@ -1019,90 +1019,103 @@ def download_file(remote_url, cache=False, show_progress=True, timeout=None):
     else:
         url_key = remote_url
 
-    try:
-        if cache:
-            # We don't need to acquire the lock here, since we are only reading
-            with _open_shelve(urlmapfn, True) as url2hash:
-                if url_key in url2hash:
-                    return url2hash[url_key]
+    url_key_list = url_key.split(',')
```

Comment:
I agree that a list seems nicer and cleaner, *and* that `ConfigItem` can be a list.  But the catch seems to be that a list-like `ConfigItem` must be specified as a list.  I just tried an example of `astropy.vo.validator.noncritical_warnings` (which is a list) and one cannot set it to a single string value.  So it seems that `IERS_A_URL` can be either a string or a list, but not both.  Is there a clean way around this?